metrics/metrics.go

```diff
@@ -72,19 +75,45 @@ func InitOrError(m interface{}, factory Factory, globalTags map[string]string) e
 				tags[tag[0]] = tag[1]
 			}
 		}
+		if bucketString := field.Tag.Get("buckets"); bucketString != "" {
```

Comment:
we need parsing for timer buckets too, right? 

we could combine all changes to the reflection factory to a separate PR, since they are non-breaking after the API change is merged. Up to you.

Reply:
Yes will handle in separate PR.

Reply:
#65